packages/slider/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hig/slider",
-  "version": "0.1.3",
+  "version": "1.0.0",
```

Comment:
A little nitpick, but I don't think you have to manually bump this to `1.0.0` as I didn't find any components that have `@hig/slider` as a dependency.